src/dice.js

```diff
@@ -0,0 +1,20 @@
+import randomjs from 'random-js';
```

Comment:
The spacing of this file should be made consistent with the rest of the project. 

To me, 2 spaces is hard to easily track which scope you're in when you're going 3 or more deep.